duckduckgo/duckduckgo-privacy-extension
# Safari UI: dark header

**Reviewer:**
@jdorweiler 

**CC: @andrey-p 

## Description:
This ports the UI changes to the search box in the header from the develop branch to the Safari branch.

Original PR to develop:
https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/76

## Steps to test this PR:
1. build this branch for safari
2. confirm header is dark, everything works as usual

![safari-dark-header](https://user-images.githubusercontent.com/1278969/35892123-b5e7a174-0b5c-11e8-9db8-51934aae0763.png)


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
